internal/utils/helper.go

```diff
@@ -71,6 +74,19 @@ func CheckPort(port int) int {
 	return port
 }
 
+func CheckTeleportStatus(port int) bool {
```

Comment:
comment - it's more `CheckIfPortOpened` or something like that. Always think that u can reuse a function so make a generic name if possible.